keyboards/nopunin10did/railroad/info.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
This file renders incorrectly, and seems to have the wrong order in "fast input" mode.

![image](https://user-images.githubusercontent.com/16520359/94865397-689f9a00-0435-11eb-8d6e-ec2603411556.png)

Reply:
I've corrected the incorrectly placed shift, and I've replaced the ISO enter with a 2x1.25, since the "ks" attribute appears not to work as described in the info.json format guide.

The order of the keys was correct until someone started rearranging everything in my files.